src/containers/PageContainer.jsx

```diff
@@ -3,6 +3,12 @@ import reactn from 'reactn';
 import PropTypes from 'prop-types';
 import { graphql } from 'gatsby';
 import { WithQAing } from './WithQAing';
+import {
+  getBlocksGroups,
+  getContents,
+  getQuestions,
+  getQuestionsWithNumbers,
+} from '../lib/utilities';
```

Comment:
methods in `PageContainer` were moved to `utilities.js` so that they can be re-used by other components implementing a block setup